src/views/components/LogConsole.mxml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0"?>
+<s:List xmlns:fx="http://ns.adobe.com/mxml/2009"
+        xmlns:s="library://ns.adobe.com/flex/spark"
+        dataProvider="{logLines}"
+        show="onAdded()"
+        creationComplete="onAdded()"
+        hide="onRemoved()"
+        contentBackgroundAlpha="0"
+        >
+    <fx:Script><![CDATA[
+        import com.heyzap.sdk.HeyzapEvent;
+        import com.heyzap.sdk.ads.BannerAd;
+        import com.heyzap.sdk.ads.IncentivizedAd;
+        import com.heyzap.sdk.ads.InterstitialAd;
+        import com.heyzap.sdk.ads.VideoAd;
+
+        import mx.collections.ArrayCollection;
+
+        import spark.core.NavigationUnit;
+
+        [Bindable]
+        private var logLines:ArrayCollection;
+
+        private function onAdded():void {
+            logLines = new ArrayCollection();
+            setListeners();
+        }
+
+        private function onRemoved():void {
+            removeListeners();
+            logLines.removeAll();
+            logLines = null;
+        }
+
+        public function showLine(string:String):void {
+            logLines.addItem(string);
```

Comment:
some sort of timestamp might be nice here for context, it's been useful before in our other test apps